src/main/relation/value_relation.cpp

```diff
@@ -66,6 +66,23 @@ ValueRelation::ValueRelation(const shared_ptr<RelationContextWrapper> &context,
 	TryBindRelation(columns);
 }
 
+ValueRelation::ValueRelation(const shared_ptr<RelationContextWrapper> &context,
+                             vector<vector<unique_ptr<ParsedExpression>>> &&expressions_p, vector<string> names_p,
+                             string alias_p)
+    : Relation(context, RelationType::VALUE_LIST_RELATION), alias(std::move(alias_p)) {
```

Comment:
Maybe the `ValueRelation` above can call this constructor instead, so that we don't need to have duplicate code?